### PR TITLE
Update to support account context on the authenticated client

### PR DESCRIPTION
**Pull request by deviantintegral**
Depends on https://github.com/Lullabot/mpx-php/pull/154

TODOs:
- [x] Update composer.json when upstream has been merged.

### CODE REVIEW

File: src/Commands/MpxImporter.php [hunk 1]
```diff
@@ -149,7 +149,7 @@ public function warmCache(string $media_type_id) {
 
     $factory = $this->dataObjectFactoryCreator->fromMediaSource($media_source);
     $query = new ObjectListQuery();
-    $request = $factory->selectRequest($query, $account);
+    $request = $factory->selectRequest($query);
```
Comment (m4olivei) [inline src/Commands/MpxImporter.php:151]: Looks like `$account` is unused in this method now. Remove definition on line 148?

**Review by m4olivei — APPROVED** [review]
Just one minor note. Feel free to address and merge!

**Comment by deviantintegral** [comment]
Testing in combination with #105 revealed an interesting bug. We weren't actually setting an account context when listening for notifications, and were only setting it when loading the videos themselves. That led to a silently discarded 404 error, and everything _looked_ to be fine. However, adding the logging in #105 showed the errors when running cron, which amusingly led to the expected behaviour of limiting imports to the configured account.

I've added a `fromAccount` factory method to simplify this, and use it in `NotificationListener`.

**Review by m4olivei — APPROVED** [review]
Static review of changes looks good 👍 . Just need to address the merge conflict, then feel free to merge.

**Pull request merged**